ILSANG/Sources/Views/Ranking/RankingItemView.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  RankingItemView.swift
+//  ILSANG
+//
+//  Created by Kim Andrew on 10/7/24.
+//
+
+import SwiftUI
+
+struct RankingItemView: View {
+    
+    var level: Int
+    var nickName: String
+    var stat: String
+    var point: Int
+    
+    var body: some View {
+        HStack (spacing: 17) {
+            RankingProfile(profileImage: nil, level: level)
+            
+            VStack (alignment: .leading, spacing: 3) {
+                Text(nickName)
+                    .font(.system(size: 15))
+                    .fontWeight(.bold)
+                Text("\(stat): \(point)p")
+                    .font(.system(size: 13))
+                    .foregroundColor(.gray400)
+            }
+            .frame(height: 40)
+            
+            Spacer()
+        }
+        .padding(.horizontal, 19)
+        .padding(.vertical, 21)
+        .frame(width: 353, alignment: .topLeading)
+        .background(.white)
+        .cornerRadius(16)
```

Comment:
```suggestion
        .padding(.horizontal, 19)
        .padding(.vertical, 21)
        .frame(maxWidth: 353, alignment: .topLeading)
        .background(.white)
        .cornerRadius(16)
        .padding(.horizontal, 20)
```

다른 해상도도 대응하면 좋을 것 같습니다!!
- width -> maxWidth로 변경
- 마지막 horizontal padding 추가

Reply:
수정 완료

Reply:
.frame(maxWidth: .infinity, alignment: .topLeading)
수정해주신거 확인했는데, `maxWidth: infinity`로 한 번 더 확인 가능할까요..

Reply:
넵!